pages/rdm_cycle/sharing.md

```diff
@@ -1,40 +1,47 @@
 ---
 title: Sharing
+contributors: [Flora D'Anna, Bert Droesbeke, Niclas Jareborg]
 keywords:
 ---
 
 ## What is Data Sharing?
-Data Sharing datasets means making datasets known and available to the whole world and society.
-Does not have to happen immediately, while you are still working on the project. Data can be shared after your scientific paper is published.
+Data Sharing means making datasets known and available to the global research community and to society at large. Data sharing can be done at any time during the research data life cycle, but at the latest, data should be made available at the time of publication of articles that use the data to make scientific conclusions. 
 
 ## Why Data Sharing is important?
-Data Management Planning will facilitate the progress of your project, as you can prepare early on. Data Management Planning will also help you to get funded, as this is a critical part of the proposal and/or the contract for many funders and a good Data Management Plan (DMP) will show and improve the feasability of your project.
+Sharing of data is a cornerstone of good science. It is ethical good research  practice to ensure that data underlying research is preserved and made available to the research community and society at large. Sharing data is a necessary prerequisite for making your research reproducible. To be useful for others, you should strive to make the shared data adhere to the [FAIR principles](). 
+
+In the EU, the 'Open Data Directive' ([Directive (EU) 2019/1024](https://eur-lex.europa.eu/legal-content/EN/TXT/?qid=1561563110433&uri=CELEX:32019L1024)) states that "_Member States shall support the availability of research data by adopting national policies and relevant actions aiming at making publicly funded research data openly available (‘open access policies’), following the principle of ‘open by default’ and compatible with the FAIR principles._"
+
+Many research funders, institutions and reputable journals/publishers now have data sharing mandates, from which you normally cannot opt out of unless there are legitimate reasons. 
+
+* Additional reasons to share your datasets:
+  - [Ten reasons to share your data.](https://www.natureindex.com/news-blog/ten-reasons-to-share-your-data)
+  - [Ask not what you can do for open data; ask what open data can do for you.](http://blogs.nature.com/naturejobs/2017/06/19/ask-not-what-you-can-do-for-open-data-ask-what-open-data-can-do-for-you/)
+
+Eventhough all data might not be possible to share openly because of ethical, legal, contractual, or intellectual property reasons, do strive to make data "_as open as possible, as closed as necessary_". 
```

Comment:
```suggestion
Even though it may not be possible to openly share all data because of ethical, legal, contractual, or intellectual property reasons, do strive to make data "_as open as possible, as closed as necessary_".
```